x-pack/plugins/lens/public/metric_visualization_plugin/metric_config_panel.test.tsx

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { ReactWrapper } from 'enzyme';
+import { mountWithIntl as mount } from 'test_utils/enzyme_helpers';
+import { MetricConfigPanel } from './metric_config_panel';
+import { DatasourcePublicAPI, DatasourceDimensionPanelProps, Operation } from '../types';
+import { State } from './types';
+import { NativeRendererProps } from '../native_renderer';
+
+describe('MetricConfigPanel', () => {
+  const dragDropContext = { dragging: undefined, setDragging: jest.fn() };
+
+  function mockDatasource(): DatasourcePublicAPI {
```

Comment:
There are exhaustive mock objects in `x-pack/plugins/lens/public/editor_frame_plugin/mocks.tsx` - should we move them to the `common` folder? They will probably get handy in a lot of cases

Reply:
Probably worthwhile. I tend to dislike too much code reuse in tests, because the tests often need to change independently of one another. But in the case of filling out big interfaces, I think the reuse is warranted.